content/content.json

```diff
@@ -99,11 +99,12 @@
       "contributor": "Rizel Scarlett"
     },
     {
-      "type": "short",
-      "thumbnail": "./img/using-goose-as-a-github-action.png",
-      "title": ""
-      "link": "Using Goose as a GitHub Action",
+      "type": "blog",
+      "thumbnail": "./img/How to Set Up Goose on Windows ",
+      "title": "How to Set Up Goose on Windows",
+      "link": "https://dev.to/lymah/how-to-set-up-goose-on-windows-29dd",
```

Comment:
love the blog post! Excellent. However, I think you're overwriting your youtube short entry. I think you meant to add another separate entry.